docs/generated/APPLICATION_CONFIG.md

```diff
@@ -323,6 +323,15 @@ Comma separated list of all GCS bucket names that all export models can use. Onl
 
 *Example value:* `bq-export-uscentral1,bq-export-useast1`
 
+### tanagra.export.shared.generateSignedUrls
```

Comment:
It would be better to flip this to `generateUnsignedUrls` so that the default is the expected prod behavior. Less chance of it getting set incorrectly and causing an outage or security issue.